client/src/components/AuditBoard/BallotAudit.tsx

```diff
@@ -38,84 +68,129 @@ const SubmitButton = styled(FormButton)`
 `
 
 interface IProps {
+  ballot: IBallot
   contests: IContest[]
   interpretations: IBallotInterpretation[]
-  interpretationsInitializedAt: Date
   setInterpretations: (interpretations: IBallotInterpretation[]) => void
-  previousBallot: () => void
   confirmSelections: (interpretations: IBallotInterpretation[]) => void
+  submitBallotNotFound: () => void
+  previousBallot: () => void
 }
 
 const BallotAudit: React.FC<IProps> = ({
```

Comment:
Much easier to review the changes to this component ignoring whitespace with [?w=1](https://github.com/votingworks/arlo/pull/1462/commits/c0f54487faaf487fd2b4792d6606bff446cd9909?w=1)!